Makefile.machine.include

```diff
@@ -37,12 +37,16 @@ BSG_MACHINE_MAX_EPA_WIDTH             = 28
 BSG_MACHINE_DRAM_SIZE_WORDS           = 536870912
 BSG_MACHINE_DRAM_BANK_SIZE_WORDS      = 134217728
 
+BSG_MACHINE_IO_REMOTE_LOAD_CAP        = 32
+BSG_MACHINE_IO_HOST_CREDITS_CAP       = 256
+BSG_MACHINE_IO_EP_MAX_OUT_CREDITS     = 16
+
```

Comment:
Can you add these to all of the machine files, and add a sentence describing what they do?